renpy/gl2/live2d.py

```diff
@@ -928,6 +928,7 @@ def render(self, width, height, st, at):
 
         zoom = self.zoom
 
+        # TODO : check this regarding the new absolute.compute
```

Comment:
The doc officially only supports floats. I don't know why integers seem technically supported. Upgrading to all/any position type is left as an exercise to the reader (who hopefully knows anything about live2D).